driver-core/src/main/java/com/datastax/driver/core/StreamIdGenerator.java

```diff
@@ -95,7 +95,7 @@ public int next() throws BusyConnectionException {
             if (id >= 0)
                 return id + (64 * j);
         }
-        throw new BusyConnectionException();
+        return -1;
```

Comment:
This is the only functional change in this pr; I had to throw BusyConnectionException outside of this class in order to have to host address.